src/trumbowyg.js

```diff
@@ -1418,7 +1418,8 @@ Object.defineProperty(jQuery.trumbowyg, 'defaultOptions', {
             var $modal = $('<div/>', {
                 class: prefix + 'modal ' + prefix + 'fixed-top'
             }).css({
-                top: t.$box.height() + t.$btnPane.height()
+                top: t.$box.offset().top + t.$btnPane.height(),
+                zIndex: 99999
```

Comment:
Why so high value?

I'm pretty sure we can set a reasonable value here.

Reply:
It's set to 99999 so that it will appear on top of the fullscreen editor, which you have set to 99999 in your CSS.

    .trumbowyg-fullscreen {
      position: fixed;
      top: 0;
      left: 0;
      width: 100%;
      height: 100%;
      margin: 0;
      padding: 0;
      z-index: 99999; }

Reply:
Oh, Okay. My bad, sorry.